trackintel/preprocessing/triplegs.py

```diff
@@ -5,89 +5,29 @@
 import pandas as pd
 from tqdm import tqdm
 
-def smoothen_triplegs(triplegs, method='douglas-peucker', tolerance=1.0):
-    """reduces number of points while retaining structure of tripleg
+def smoothen_triplegs(triplegs, tolerance=1.0):
+    """Reduces number of points while retaining structure of tripleg
+    
     Parameters
     ----------
-    triplegs: shapely file
-        triplegs to be reduced
-    method: method used to smoothen
-        only the douglas-peucker method is available so far
+    triplegs: GeoDataFrame
+        triplegs to be simplified
+        
     tolerance: float
         a higher tolerance removes more points; the units of tolerance are the same as the projection of the input geometry
-    """
-    input_copy = copy.deepcopy(triplegs)
-    origin_geom = input_copy.geom
-    simplified_geom = origin_geom.simplify(tolerance, preserve_topology=False)
-    input_copy.geom = simplified_geom
-
-    return input_copy
-
-def _temp_trip_stack_has_tripleg(temp_trip_stack):
-    """
-    Check if a trip has at least 1 tripleg
-    Parameters
-    ----------
-        temp_trip_stack : list
-                    list of dictionary like elements (either pandas series or
-                    python dictionary). Contains all elements
-                    that will be aggregated into a trip
-
+    
     Returns
-    -------
-    Bool
-    """
-
-    has_tripleg = False
-    for row in temp_trip_stack:
-        if row['type'] == 'tripleg':
-            has_tripleg = True
-            break
-
-    return has_tripleg
-
-
-def _create_trip_from_stack(temp_trip_stack, origin_activity, destination_activity):
```

Comment:
I think we should agree on either having the helper functions on the top or having the helper functions at the bottom and then add it as a convention (e.g., in the readme). 
I just had a look and [scikit ](https://github.com/scikit-learn/scikit-learn/blob/b3ea3ed6a/sklearn/manifold/_mds.py)does the helper function first, while [pandas](https://github.com/pandas-dev/pandas/blob/v1.2.0/pandas/io/date_converters.py)and geopandas have it at the bottom, so it is really up to us. I am in favor of having the helpers at the top but I am really happy with both as long as it is consistent and written down somewhere.

Reply:
Currently, I put all helper functions at the bottom for readability. I did not find a good place to document this though.

Reply:
We could have a "conventions for developers" section or something like that